tests/khalendar_test.py

```diff
@@ -107,10 +107,10 @@ def test_list(self, coll_vdirs):
         event = Event.fromString(event_today, calendar=cal1, locale=LOCALE_BERLIN)
         coll.new(event)
         hrefs = sorted(href for href, etag in coll._backend.list(cal1))
-        assert set(str(coll.get_event(href, calendar=cal1).uid) for href in hrefs) == set((
+        assert {str(coll.get_event(href, calendar=cal1).uid) for href in hrefs} == {(
```

Comment:
This one works but changes the meaning.
```suggestion
        assert {str(coll.get_event(href, calendar=cal1).uid) for href in hrefs} == {
```

Reply:
It does?

Reply:
```py
>>> set(('uid3@host1.com', 'V042MJ8B3SJNFXQOJL6P53OFMHJE8Z3VZWOU'))
{'uid3@host1.com', 'V042MJ8B3SJNFXQOJL6P53OFMHJE8Z3VZWOU'}
>>> {('uid3@host1.com', 'V042MJ8B3SJNFXQOJL6P53OFMHJE8Z3VZWOU')}
{('uid3@host1.com', 'V042MJ8B3SJNFXQOJL6P53OFMHJE8Z3VZWOU')}
>>> {'uid3@host1.com', 'V042MJ8B3SJNFXQOJL6P53OFMHJE8Z3VZWOU'}
{'uid3@host1.com', 'V042MJ8B3SJNFXQOJL6P53OFMHJE8Z3VZWOU'}
```

Reply:
The first one looks fine, just the one on the following lines (that starts here)

Reply:
Thanks for finding this one too!